roadmap/implementers-guide/src/types/runtime.md

```diff
@@ -39,7 +39,23 @@ struct HostConfiguration {
 	/// Total size of messages allowed in the parachain -> relay-chain message queue before which
 	/// no further messages may be added to it. If it exceeds this then the queue may contain only
 	/// a single message.
-	pub watermark_upward_queue_size: u32,
+	pub max_upward_queue_size: u32,
+	/// The amount of weight we wish to devote to the processing the dispatchable upward messages
+	/// stage.
+	///
+	/// NOTE that this is a soft limit and could be exceeded.
```

Comment:
Should we consider removing notes and making it part of the regular text?
```suggestion
	/// It is a soft limit and could be exceeded.
```

Reply:
I believe that this note is worth attention hence the tag. I am not married with this particular tag however. Not sure what is a common way to do a similar thing in the guide

Reply:
Frankly, I have no strong opinion about it, I just did not see much usage in other parts of the guide.